docs/api_reference/messages_api/post_send_messages.md

```diff
@@ -143,3 +147,109 @@ curl -X POST "https://api.callbell.eu/v1/messages/send" \
     }
   }'
 ```
+
+## Send Template Messages
+
+:::info
+This is only available for accounts using the official **WhatsApp Business API** integration.
+:::
+
+You can use the API to send an approved [Template](/api_reference/object_types/template) Message
+In order to send template messages `template_uuid` and `optin_contact` **must** be present in the payload.
```

Comment:
Probably this could be warning banner?

```suggestion
:::warn
In order to send template messages `template_uuid` and `optin_contact` **must** be present in the payload.
:::
```